portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,39 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 90%;
 }
 
-#greeting-container {
+#quote-container {
   margin-top: 20px;
 }
+a:link {
+  color: black;
+}
+.image1 {
+  position: absolute;
+  top: 10px;
+  right: 50%;
+  width:15%; }
+
+h1 {
+  margin-left: auto;
+  margin-right: auto;
+  font-size: 4.5em;
+  font-family: Arial, Helvetica, sans-serif;
+}
+a {
+  font-size: 1.5em;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+p{
```

Comment:
Minor nit: try to be consistent with spaces and indentation.
So here, add a space after between `p {` and generally align the individual rules, though in this case probably combine to the single font property.